site2/docs/reference-cli-tools.md

```diff
@@ -448,6 +448,7 @@ Options
 |`-ss`, `--subscriptions`|A list of subscriptions to consume on (e.g. sub1,sub2)|sub|
 |`-st`, `--subscription-type`|Subscriber type. Possible values are Exclusive, Shared, Failover, Key_Shared.|Exclusive|
 |`-sp`, `--subscription-position`|Subscriber position. Possible values are Latest, Earliest.|Latest|
+|`-time`, `--test-duration`|Test duration in secs. If <= 0, it will keep consuming|0|
```

Comment:
```suggestion
|`-time`, `--test-duration`|Test duration (in seconds). If this value is less than or equal to 0, it keeps consuming messages.|0|
```

Please check all occurrences and keep consistent.

Reply:
Thanks for your contribution. Does this affect only master or other versioned docs? 
If latter, could you please help update all affected versions? Thanks

Reply:
1. The document has been checked and updated, PTAL again @Anonymitaet , thx!
2. This affects only master or other versioned docs.